frontend/src/scenes/onboarding/sdks/sdksLogic.tsx

```diff
@@ -118,6 +119,20 @@ export const sdksLogic = kea<sdksLogicType>([
                 return Object.keys(availableSDKInstructionsMap).length > 5 && sourceOptions.length > 2
             },
         ],
+        combinedSnippetAndLiveEventsHosts: [
+            (selectors) => [selectors.snippetHosts, selectors.eventHosts],
+            (snippetHosts: string[], eventHosts: string[]): string[] => {
+                const combinedSnippetAndLiveEventsHosts = snippetHosts
+                for (const host of eventHosts) {
+                    const hostProtocol = new URL(host).protocol
+                    const currentProtocol = window.location.protocol
+                    if (hostProtocol === currentProtocol && !combinedSnippetAndLiveEventsHosts.includes(host)) {
```

Comment:
Idea: you could use a `Set` here instead which would make the logic a little cleaner.

Reply:
My editor complained about the Set so I did it this way